guides/source/action_view_helpers.md

```diff
@@ -257,53 +303,65 @@ atom_feed do |feed|
 end
 ```
 
-See the [API Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/AtomFeedHelper.html#method-i-atom_feed) for more information.
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/AtomFeedHelper.html#method-i-atom_feed)
+for more information.
 
-### BenchmarkHelper
```

Comment:
Calling this out, it seems that the benchmark helper method got deprecated.

Reply:
Sorry, deprecated or moved.

https://apidock.com/rails/ActionView/Helpers/BenchmarkHelper/benchmark

<img width="771" alt="Screenshot 2024-03-19 at 11 34 55" src="https://github.com/Ridhwana/rails/assets/2786819/d01a99ba-19cf-4233-ada1-eb4ba7a94173">

And I'm looking at https://github.com/rails/rails/tree/5411787a155e2d1c633a74314edc164d5b300c34/actionview/lib/action_view/helpers and the Benchmark Helper isn't part of the code folder.

Reply:
That's some good context! Thank you! In that case, perhaps we can continue showing Benchmark Helpers in the ActionView documentation and add a note that this helper is a part of Active Support and exists across multiple modules like controllers, helpers, views, etc. 

I'll update it based on this but let me know if you had something different in mind?